### PR TITLE
IAP-3857: API Portal docker related docs ...

### PR DESCRIPTION
**Pull request by kvedenskii**
https://deploy-preview-1529--axway-open-docs.netlify.app/docs/apim_installation/apiportal_docker/

update run API Portal container page:
  * update env variables list
  * add image self-documentation section
  * add exposed ports section
  * add volume-based configuration section
  * add upgrade section

update build API Portal image page:
  * rewrite docker build page

other changes:
  * create docker compose page
  * create upgrade page
  * change pages weight

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md [hunk 1]
```diff
@@ -28,6 +28,20 @@ The following are the recommended hardware disk space and memory requirements fo
 * 100 GB or more disk space
 * 8 GB or more RAM
 
+## API Portal image self-documentation
+
+To see API Portal image help message run:
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md:33]: @kvedenskii (To see API Portal image help message run) -> is this going to print like a **sentence** in the terminal?

**Comment by dginevaAxway** [comment]
Good from my side. @astoyanovAxway  please review it also.

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_compose.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## About Docker Compose
+
+Docker Compose is a simple way to run the whole API Portal solution with a single command. By using the `docker-compose.yml` file, provided with the API Portal Docker sample package, you get the following services preconfigured, ready to be used with API Portal:
+
+* MariaDB
+* Redis
+* API Portal
+
+The `docker-compose.yml` file does not include API Manager and ClamAV configurations, you must configure them in API Portal separately. While API Manager is required for you to leverage your API Portal, ClamAV is an optional security tool.
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_docker/docker_portal_compose.md:23]: ClamAV -> This Antivirus was [deprecated](https://axway-open-docs.netlify.app/docs/apim_relnotes/20200730_apimgr_relnotes/#deprecated-features) from November update, so why don't you remove this from the instructions?

Comment (kvedenskii) [reply content/en/docs/apim_installation/apiportal_docker/docker_portal_compose.md:23]: @andreamussap
It's deprecated in API Gateway. For API Portal it's currently the only files scanning solution.

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md (hunk 1 above)
Comment (kvedenskii) [reply content/en/docs/apim_installation/apiportal_docker/docker_portal_run_image.md]: @andreamussap 
Yes, it prints out a cheat sheet on how to use the image.

**Comment by andreamussap** [comment]
Label `More Info Needed` - We're discussing this more on MS-Teams

**Pull request merged**